pkg/apiserver/registry/security/credentialsbinding/credentialsbinding_suite_test.go

```diff
@@ -0,0 +1,20 @@
+// SPDX-FileCopyrightText: SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package credentialsbinding_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/gardener/gardener/pkg/apiserver/features"
+)
+
+func TestCredentialsBinding(t *testing.T) {
+	features.RegisterFeatureGates()
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Registry Security CredentialsBinding Suite")
```

Comment:
Nit: I would have expected to have something along with the package `apiserver` in the name of the suite.